pwiz_tools/Skyline/Properties/Resources.Designer.cs

```diff
@@ -34076,15 +34186,15 @@ public class Resources {
         }
         
         /// <summary>

```

Comment:
ugh, would really like to know why these comments all changed. Really muddies the waters.